datasets/index.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: single
+author_profile: false
+title: Zarr Datasets
+sidebar:
+  title: "Content"
+  nav: sidebar
+---
+
+<font size="4">
+According to the <a href="https://zarr.dev/adopters/">website</a>, Zarr is
+utilised by a variety of research groups, universities, organisations, and
+open-source communities.<br><br>
+
+We have compiled a list of public datasets for the community to use.<br><br>
+</font>
+
+| Dataset                                                                                                           | Hosting               | Size  |
+| ----------------------------------------------------------------------------------------------------------------- | --------------------- | ----- |
+| [CMIP6](https://console.cloud.google.com/marketplace/details/noaa-public/cmip6)                                   | Google Cloud          | 1 PB  |
+| [IDR (OME-NGFF)](https://idr.github.io/ome-ngff-samples/)                                                         | EBI                   | 3 TB  |
+| [Microsoft Planetary Computer](https://planetarycomputer.microsoft.com/catalog?filter=zarr)                       | Microsoft Azure       |       |
+| [MUR Sea Surface Temperature](https://registry.opendata.aws/mur)                                                  | AWS Open Data Program | 54 TB |
+| [NASA POWER](https://registry.opendata.aws/nasa-power/)                                                           | AWS Open Data Program |       |
+| [National Water Model](https://discourse.pangeo.io/t/the-national-water-model-reanalysis-zarr-dataset-on-aws/1449)| Amazon AWS            | 5 TB  |
+| [NGFF](https://ngff.openmicroscopy.org/data/index.html)                                                           | Various providers     |       |
```

Comment:
Maybe this could be in another section as "other catalogs".

Reply:
Thanks for the good suggestion! 

How about we keep [NGFF](https://ngff.openmicroscopy.org/data/index.html) where it is now and separate it out into 'Catalogs' sections once we have a few more examples?

Reply:
Then I'd rather just remove it for the moment. We can always add it later.